python/paddle/distributed/parallel.py

```diff
@@ -402,4 +423,14 @@ def get_world_size():
             print("The world_size is %d" % dist.get_world_size())
             # The world_size is 4
     """
+    if in_dygraph_mode():
+        # NOTE(liyurui): We need to ensure that this API is also correct in
+        # non distributed mode.
+        if collective.is_initialized():
+            group = collective._get_default_group() if group is None else group
+            return group.world_size
+        else:
+            return 1
```

Comment:
Raise exception.

Reply:
This need to add is_initialize() api before raise exception